template.rb

```diff
@@ -273,10 +279,6 @@ def gemfile_requirement(name)
   req && req.tr("'", '"').strip.sub(/^,\s*"/, ', "')
 end
 
-def git_repo_specified?
-  TEMPLATE_CONFIG.git_repo_url != "skip" && !TEMPLATE_CONFIG.git_repo_url.strip.empty?
-end
-
```

Comment:
Our templates are invoked within a Rails environment so we can use things like `#present?`